crates/api_crud/src/custom_emoji/list.rs

```diff
@@ -0,0 +1,26 @@
+use actix_web::web::{Data, Json, Query};
+use lemmy_api_common::{
+  context::LemmyContext,
+  custom_emoji::{ListCustomEmojis, ListCustomEmojisResponse},
+};
+use lemmy_db_views::structs::{CustomEmojiView, LocalUserView, SiteView};
+use lemmy_utils::error::LemmyError;
+
+#[tracing::instrument(skip(context))]
+pub async fn list_custom_emojis(
+  data: Query<ListCustomEmojis>,
+  local_user_view: Option<LocalUserView>,
+  context: Data<LemmyContext>,
+) -> Result<Json<ListCustomEmojisResponse>, LemmyError> {
+  let local_site = SiteView::read_local(&mut context.pool()).await?;
+  let custom_emojis = CustomEmojiView::list(
+    &mut context.pool(),
+    local_site.local_site.id,
+    data.page,
+    data.limit,
+  )
+  .await
+  .map_err(|e| anyhow::anyhow!("Failed to construct custom emojis response: {e}"))?;
```

Comment:
I think `.context()` makes more sense here, [example](https://github.com/LemmyNet/lemmy/blob/main/crates/federate/src/worker.rs#L133).

Reply:
Removed the message